modules/io/adaptors/read_hive_orc.py

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Alibaba Group Holding Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import vineyard
+import pyorc
+import pyarrow as pa
+import sys
+import json
+
+from urllib.parse import urlparse
+from hdfs3 import HDFileSystem
+from vineyard.io.dataframe import DataframeStreamBuilder
+
+
```

Comment:
I'm if it is possible to write a note here, e.g., a note about how an hive table are originaized as a batch of files under a. HDFS directory, how the hive's schema been mapped to orc's schema, the spec of orc file chunk size (by rows, by columns, or by bytes).

Such a note will be helpful for others to understand how vineyard's IO interacts with hive tables, and avoid a duplicate learning process.